### PR TITLE
Get bitmaps working in Linux framebuffer in direct mode

### LINKED ISSUE #1959: framebuffer bitmaps don't work in direct mode
Direct mode wants to blit the faux-`ncvisual` itself. This leads to serious problems with framebuffer bitmaps, given that they're directly written to the `mmap()ped` framebuffer, and not a true glyph. This might be really difficult to solve for direct mode, so I'm thinking maybe we just disable them there, idk. I'd rather not do that, but the current behavior can't stand.

### PR DESCRIPTION
**Pull request by dankamongmen**
This more or less gets bitmaps working in Linux when using Direct Mode. Closes #1959.

### CODE REVIEW

File: src/lib/iterm.c [hunk 1]
```diff
@@ -26,9 +26,12 @@ int iterm_rebuild(sprixel* s, int ycell, int xcell, uint8_t* auxvec){
 }
 
 // spit out the control sequence and data.
-int iterm_draw(const ncpile *p, sprixel* s, FILE* out, int y, int x){
-  if(goto_location(p->nc, out, y, x)){
-    return -1;
+int iterm_draw(const tinfo* ti, const ncpile *p, sprixel* s, FILE* out, int y, int x){
+  (void)ti;
```
Comment (dankamongmen) [inline src/lib/iterm.c:30]: this ought fix bitmap placement on iTerm, which was apparently broken. @michaelsbradleyjr does that track with what you've seen? before this change, they always ought have been emitted at the current cursor location.

Comment (dankamongmen) [reply src/lib/iterm.c:30]: nevermind, it already had the `goto_location()` and thus probably already worked.

**Review by dankamongmen — COMMENTED** [review]
this isn't as messy as i recall it, and i see no immediate problems. merging.

File: src/lib/iterm.c (hunk 1 above)
Comment (michaelsbradleyjr) [reply src/lib/iterm.c:30]: In my experience it was already working.

**Pull request merged**